config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  root to: "home#index"
```

Comment:
[REQUIRED] Kindly fix the issue with confirming emails because I am getting an error when I click the confirmation link
![image](https://github.com/chiztechnology/budget-app/assets/62938735/4f5a73e0-faa0-4906-a9b8-8acc27a21d36)
Fixing this is necessary for proper review